docs/run-random-beacon.adoc

```diff
@@ -0,0 +1,392 @@
+:toc: macro
+
+= Run Random Beacon
+
+:icons: font
+:numbered:
+toc::[]
+
+== System Recommendations
+
+- It is paramount that Keep nodes remain available to the Keep network. We encourage a stable and redundant internet connection.
+- A connection to a production grade self-hosted or third party Ethereum node deployment.
+- Persistent storage that will survive a VM or container rotation.
```

Comment:
Worth noting each instance should have its own ethereum key, no two or more instances with the same key should be running.

Reply:
Yep, we do that in Deployment section but will look at collapsing that.